h5py/tests/test_file2.py

```diff
@@ -273,3 +275,36 @@ def test_no_track_order(self):
         self.populate(f)
         self.assertEqual(list(f),
                          sorted([str(i) for i in range(100)]))
+
+
+class TestFileMetaBlockSize(TestCase):
+
+    """
+        Feature: The minimum metadata block size can be set.
+    """
+
+    def test_file_create_with_meta_block_size(self):
+        meta_block_size = 4096
+        with File(
+            self.mktemp(), 'w',
+            meta_block_size=meta_block_size
+        ) as f:
+            self.assertTrue(f)
```

Comment:
This is probably superfluous.

Reply:
Removed in [ae9b8d4](https://github.com/h5py/h5py/pull/2106/commits/ae9b8d4e4a27536dbd00238f9f8c5a3e1dabf039)